### DISCUSSION TITLE
Contact page

### DISCUSSION THREAD

**Discussion by michelle-rago** [GSA-TTS/federal-website-standards#183] [Draft]
We’re researching a [potential standard on contact information for federal agency websites](https://standards.digital.gov/standards/contact-research/). Research questions include:

- Where do users expect to find contact information on a page on desktop and mobile? 
- Are users able to find the contact link as easily in the header, footer, and mobile menu? 
- What kind of contact information do users expect to see (at a minimum)?

Do you have research about providing contact information on a federal government website?

**Comment by michelle-rago** [comment]
The standard for agency contact pages has moved to the draft stage. [Review the draft](https://standards.digital.gov/standards/contact-page/). Give us feedback here or via email at website.standards@gsa.gov.

**Comment by PBrennan81** [comment]
Sorry if this is a little rambley...

I think the connect with us options example might be too much for an office to manage and info overload for the end user. Some offices might be resourced to monitor web content, social media, email inquiries etc..., but agencies/business lines should really attempt to streamline the most efficient way of exchanging information. I suggest starting with one passive communication channel (web content) and one active (email or form) and then consider other channels to be advanced. That way a user across the government knows they have the opportunity to find the information independently or reach someone who can support them. You may want to be as prescriptive to say, use a form (instead of email contacts) when possible so that the experience is uniform.

I didn't see examples of setting response times.

I don't think you should use an example that repeats the domain information in the local section and should also consider plain language. CFPB in the local-part of the email is redundant and 508 is technical. Instead use an exmaple like accessibility@cfpb.gov

Also, consider talking about the strategy for email communications. Agencies should attempt to create as few email addresses as possible to reduce barriers to entry. Whoever receives that email should then be able to decipher the need and route it to the correct person for response. That means a dedicated role is needed to monitor, route and respond to emails that come in and a knowledge base might be needed to support consistent responses and routing. In GSA leasing, we recently consolidated our email to leasing@gsa.gov. When we speak to the public or post contact information, we attempt to convey that single source to make it easy for the public to get in touch with our office.

Part of the strategy could be to create a generic account: contact@cfpb.gov
and specific functions within the organizations for more targeted help: accesibility@cfpb.gov, eeo@cfpb.gov, etc...

The hard part about all of this is making sure the structure is in place to make this sustainable.

**Reply by rphillippi** [reply]
Hi, sorry for the delay.  I've read over your comment and I am not sure how to respond. I hope the information below helps. 

Standards are the base that an agency should strive to meet. The basics for a contact page that an agency needs to have on the page is in the acceptance criteria. For example, "Your contact page must include at least one of the following options: email address, contact form, phone number". 

On the CFPB page, under "submit a compliant", they have said "it can take up to 15 days to get a response". That would be an example of a response time. 

As for the basic email, are you asking us to create a standard that it should be one email and then routed? I don't believe that's the intention of the standards. Can you say more? I'm not sure if I have answered your question.
